nsqd/statsd.go

```diff
@@ -80,6 +86,10 @@ func (n *NSQD) statsdLoop() {
 				}
 
 				for _, channel := range topic.Channels {
+					if excludeEphemeral && strings.HasSuffix(channel.ChannelName, "#ephemeral") {
```

Comment:
I think you can just use `channel.ephemeral` and `topic.ephemeral` boolean properties

Reply:
These are `TopicStats` and `ChannelStats` structs respectively so those don't have the `ephemeral` field which is on the actual `Topic` and `Channel` struct.

We do a number of other suffix matches to find this so this seemed like the best path vs expanding scope to include in `TopicStats` and `ChannelStats` as those represent response structs from stats endpoints.

Reply:
ah got it